src/persist/src/postgres.rs

```diff
@@ -247,8 +253,12 @@ impl PostgresConsensus {
         // See: https://www.cockroachlabs.com/docs/stable/configure-zone.html#variables
         client
             .batch_execute(&format!(
-                "{}; {}",
```

Comment:
Looks like this old `;` was redundant, but just wanted to point it out in case removing it wasn't intentional!

Reply:
yup noticed that it was redundant while i was in there